src/OpenTelemetry.Instrumentation.AWSLambda/Implementation/AWSLambdaHttpUtils.cs

```diff
@@ -34,10 +34,10 @@ internal class AWSLambdaHttpUtils
     {
         var tags = new List<KeyValuePair<string, object>>();
 
-        string httpScheme = null;
-        string httpTarget = null;
-        string httpMethod = null;
-        string hostName = null;
+        string? httpScheme = null;
+        string? httpTarget = null;
+        string? httpMethod = null;
+        string? hostName = null;
         int? hostPort = null;
 
         switch (input)
```

Comment:
Add a `default: return` to skip the AddTagIfNotNull in these cases?

Reply:
Added.